docs/sources/whatsnew/whats-new-in-v9-1.md

```diff
@@ -0,0 +1,188 @@
+
+## OSS
+
+### Auth and security
+
+#### Grafana Service Accounts are generally available
+
+Service accounts are an evolution in machine access within Grafana. You can create multiple API tokens per service account with independent expiration dates, and temporarily disable a service account without deleting it. These benefits make Service Accounts a more flexible way for Terraform and other apps to authenticate with Grafana.
+
+Service accounts also work with [role-based access control]({{< relref "../enterprise/access-control/" >}}) in [Grafana Enterprise]({{< relref "../enterprise/" >}}): you can improve security by granting service accounts specific roles to limit the functions they can perform. Service accounts have been in beta since Grafana v8.5, and during that time we've improved the UI and migration path from API keys, and made it possible to add service accounts to teams and inherit team permissions. To learn more about Service Accounts, see the [documentation]({{< relref "../administration/service-accounts" >}}).
+
+{{< figure src="/static/img/docs/service-accounts/add-service-account-token-9-1.png" max-width="750px" caption="Adding a service account token" >}}
+
+#### JWT URL embedding
+
+You can now easily embed Grafana in other applications by adding a JWT token directly in the Grafana's URL, for example, it can look like `https://example.grafana.net/dashboard/uuid?aut_token=<jwt_token>`. When the JWT token is passed through the request URL to Grafana, Grafana will validate and authenticate the token linked to a specific user, allowing access to dashboards which that user can view. To see JWT URL embedding in action, see the [sample project](https://github.com/grafana/grafana-iframe-oauth-sample).
+
+{{< figure src="/static/img/docs/dashboards/jwt-url-embedding-9-1.png" max-width="750px" caption="A JWT token used to embed Grafana" >}}
+
+#### Organization role mapping for GitHub OAuth2 authentication
+
+You can now use GitHub OAuth2 to map users or teams to specific [Grafana organization roles]({{< relref "../administration/roles-and-permissions/#organization-roles" >}}) by using `role_attribute_path` configuration option. Grafana will use [JMESPath](https://jmespath.org/examples.html) for path lookup and role mapping. For more information, see the [documentation]({{< relref "../setup-grafana/configure-security/configure-authentication/github/#map-roles" }}).
+
+{{< figure src="/static/img/docs/permissions/org-role-mapping-github-9-1.png" max-width="750px" caption="Configuring GitHub OAuth2 authentication with role mapping" >}}
+
+### Search and navigation
+
+#### (Beta) Panel title search and search improvements
+
+We've improved the performance of searching by panel title. If a panel's title matches your search query, it will be displayed in the search results. This feature will be rolled out to Grafana Cloud users over the course of several weeks, or can be accessed by enabling the panelTitleSearch feature toggle.
```

Comment:
```suggestion
We've improved the performance of searching by panel title. If a panel's title matches your search query, it will be displayed in the search results. This feature will be rolled out to Grafana Cloud users over the course of several weeks, or can be accessed by enabling the `panelTitleSearch` feature toggle.
```